smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/OperationInputBodyMiddleware.kt

```diff
@@ -137,9 +148,12 @@ class OperationInputBodyMiddleware(
         )
     }
 
-    private fun addEventStreamMiddleware(writer: SwiftWriter, operationStackName: String, inputSymbol: Symbol, outputSymbol: Symbol, payloadSymbol: Symbol, keyPath: String, defaultBody: String) {
+    private fun addEventStreamMiddleware(writer: SwiftWriter, operationStackName: String, inputSymbol: Symbol, outputSymbol: Symbol, payloadSymbol: Symbol, keyPath: String, defaultBody: String, sendInitialRequest: Boolean) {
+        if (sendInitialRequest) {
+            writer.write("let initialRequestMessage = try input.makeInitialRequestMessage(encoder: encoder)")
```

Comment:
Does `makeInitialRequestMessage(encoder: encoder)` get generated somewhere?

Reply:
Moved the codegen for this method from aws-sdk-swift to smithy-swift as pointed out in offline discussion, given initial request is part of the generic Smithy specification.